package-kafka-serde-tools/pom.xml

```diff
@@ -77,7 +77,7 @@
                         <descriptor>src/assembly/development.xml</descriptor>
                         <descriptor>src/assembly/package.xml</descriptor>
                     </descriptors>
-                    <attach>false</attach>
+                    <!-- <attach>false</attach> -->
```

Comment:
we should just delete this instead of commenting. otherwise it's confusing why it is even there